aten/src/ATen/native/Convolution.cpp

```diff
@@ -381,7 +380,7 @@ static void check_shape_forward(const at::Tensor& input,
 
   TORCH_CHECK(!params.is_padding_neg(), "negative padding is not supported");
   TORCH_CHECK(!params.is_output_padding_neg(), "negative output_padding is not supported");
-  TORCH_CHECK(!params.is_stride_neg(), "negative stride is not supported");
+  TORCH_CHECK(!params.is_stride_nonpos(), "non positive stride is not supported");
```

Comment:
nit: maybe "non-positive" instead of "non positive"?

Reply:
done